### PR TITLE
Completed issue-1-create-an-entry.

### PR DESCRIPTION
**Pull request by Edward-J-Kim**
https://github.com/Edward-J-Kim/code-journal/assets/126144853/1c59ecba-9bf2-447d-ab17-42f400cbe07f

### CODE REVIEW

File: css/styles.css [hunk 1]
```diff
@@ -0,0 +1,48 @@
+* {
+  font-family: sans-serif;
+  padding: 5px;
```
Comment (shawnkost) [inline css/styles.css]: Remove this. It's applied padding to every single element, which is causing a white gap on top of the navbar and around the sides

File: css/styles.css [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+}
+
+label {
+  display: block;
+}
+
+input,
+textarea {
+  width: 100%;
+}
```
Comment (shawnkost) [inline css/styles.css]: These elements should have some more space between them. Give these a `margin-bottom` to space them out a bit more

File: css/styles.css [hunk 3]
```diff
@@ -0,0 +1,48 @@
+* {
+  font-family: sans-serif;
```
Comment (shawnkost) [inline css/styles.css]: We should be using 2 different fonts in this project. You'll need to import these from Google fonts. We did this in `css-fonts`.

1. Open Sans 
2. Proza Libre (for the headings)

**Review by shawnkost — CHANGES_REQUESTED** [review]
Nicely done Edward! A couple changes here

File: css/styles.css (hunk 3 above)
Comment (Edward-J-Kim) [reply css/styles.css]: Screenshot 2023-05-31 at 9 39 43 PM

updated :)

**Review by shawnkost — APPROVED** [review]
👍🏻

**Pull request merged**